packages/makeup-active-descendant/dist/cjs/index.js

```diff
@@ -45,7 +45,12 @@ var defaultOptions = {
   autoReset: -1,
   autoScroll: false,
   axis: 'both',
-  ignoreButtons: false
+  ignoreButtons: false,
+
+  /** @type {{[attr: string]: unknown}} */
```

Comment:
Just confirming, are these annotations following this spec: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html or something else?

Reply:
These are following the typescript-based JSDoc spec, yes

Reply:
Thanks!